components/error_code/src/engine.rs

```diff
@@ -7,7 +7,7 @@ define_error_codes!(
     NOT_IN_RANGE => ("NotInRange", "", ""),
     PROTOBUF => ("Protobuf", "", ""),
     IO => ("IO", "", ""),
-    CF_NAME => ("CFName", "", ""),
+    CF_NAME => ("CfName", "", ""),
```

Comment:
Will the change affect external components?

Reply:
I don't know. @5kbpers